pkg/secrets/repositoryserver/gcp_location.go

```diff
@@ -0,0 +1,17 @@
+package repositoryserver
+
+import v1 "k8s.io/api/core/v1"
+
+type GCP struct {
+	storageLocation *v1.Secret
+}
+
+func NewGCPLocation(secret *v1.Secret) *GCP {
+	return &GCP{
+		storageLocation: secret,
+	}
+}
+
+func (l *GCP) ValidateSecret() error {
+	return nil
```

Comment:
Here as well, I believe we need validation logic.

Reply:
Added